lib/Command/ExpirePassword.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * @author Jörn Friedrich Dreyer <jfd@butonic.de>
+ *
+ * @copyright Copyright (c) 2018, ownCloud GmbH
+ * @license GPL-2.0
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\PasswordPolicy\Command;
+
+use OCP\IConfig;
+use OCP\IUserManager;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+
+class ExpirePassword extends Command {
+	/** @var \OCP\IConfig */
+	private $config;
+
+	/** @var \OCP\IUserManager */
+	protected $userManager;
+
+	/**
+	 * @param IConfig $config
+	 * @param IUserManager $userManager
+	 */
+	public function __construct(IConfig $config, IUserManager $userManager) {
+		parent::__construct();
+		$this->config = $config;
+		$this->userManager = $userManager;
+	}
+
+	protected function configure() {
+		$this
+			->setName('user:expire-password')
+			->setDescription('expire a password for a user')
+			->addArgument(
+					'uid',
+					InputArgument::REQUIRED,
+					'the uid of the user in ownCloud'
+				     )
+			->addArgument(
+					'expiredate',
+					InputArgument::OPTIONAL,
+					'expire the password at the given date and time, eg. "2019-01-01 14:00:00 CET"',
```

Comment:
How about '_The date and time when the password expires. e.g., "2019-01-01 14:00:00 CET"._'

Reply:
`expires. e.g., ` dot before e.g.?